best-practices/tagging/index.html

```diff
@@ -401,14 +420,26 @@ <h4>Lists</h4>
 					<dt>Element(s):</dt>
 					<dd>
 						<p>[^ol^], [^ul^], [^dl^], [^li^]</p>
+
+						<p>Note that <code>ol</code> is preferred for <code>exercise</code>, <code>poetry</code>, and <code>generic</code>, while <code>dl</code> is preferred for <code>glossary</code>, <code>index</code>, and <code>toc</code>. <code>ul</code> can be used for basic lists.</p>
```

Comment:
Is this correct that dl is preferred for table of contents and indexes? Example 24 shows toc on an ol, for example, which is how epub also requires the table of contents be done in the navigation document. I'm not sure I can visualize how to use dd/dt pairs for these.

Reply:
Fixed. There is no reason to use dl for toc. I don't know what I was thinking when I put it there. I was probably just confused.